roles/repositories/README.md

```diff
@@ -0,0 +1,108 @@
+theforeman.foreman.repositories
+===============================
+
+This role defines Products and enabled Red Hat Repositories and Custom Repositories.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+- `products`: List of products to enable and synchronize.
+  Each product is represented as a dictionary and can include `repository_sets` which represent Red Hat Repositories and should be used when the product name matches an existing Red Hat Product.
+  Each repository set must have a `name` and should specify the `basearch` and/or `releasever` only when multiple versions are available for that product.
+  Alternatively, a custom product can have associated `repositories` which represent custom repositories.
+  Custom repositories are required to have a `name`, `url`, and `content_type`; they may require additional fields.
+  A variety of examples are demonstrated in the data structure below:
+
+```yaml
+products:
+  - name: Red Hat Enterprise Linux Server
+    repository_sets:
+      - name: Red Hat Enterprise Linux 7 Server (RPMs)
+        basearch: x86_64
+        releasever: 7Server
+      - name: Red Hat Enterprise Linux 6 Server (RPMs)
+        basearch: x86_64
+        releasever: 6Server
+      - name: Red Hat Enterprise Linux 7 Server - Extras (RPMs)
+        basearch: x86_64
+      - name: Red Hat Enterprise Linux 7 Server - Optional (RPMs)
+        basearch: x86_64
+  - name: Red Hat Software Collections (for RHEL Server)
+    repository_sets:
+      - name: Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server
+        basearch: x86_64
+        releasever: 7Server
+      - name: Red Hat Software Collections RPMs for Red Hat Enterprise Linux 6 Server
+        basearch: x86_64
+        releasever: 6Server
+  - name: Red Hat Enterprise Linux for x86_64
+    repository_sets:
+      - name: Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+        releasever: 8
+      - name: Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
+        releasever: 8
+  - name: CentOS 8
+    repositories:
+      - name: BaseOS x86_64
+        content_type: yum
+        url: http://mirror.centos.org/centos/8/BaseOS/x86_64/os/
```

Comment:
Mind adding a second repostitory to this example for AppStream to show why you would want to do a `CentOS 8` product instead of a `CentOS` product with 7 and 8 in it?

Reply:
Done.